docs/configure-and-manage/configure/consensus-protocols/raft.md

```diff
@@ -45,3 +51,16 @@ By default, GoQuorum listens on port 50400 for the Raft transport. Use the
 
 By default, the number of peers is 25. Use the `--maxpeers N` command line option to configure the
 maximum number of peers where N is expected size of the cluster.
+
+## Migrate from Raft to another consensus protocol
+
+Migrating a network using Raft to a consensus mechanism suitable for production such as [QBFT](qbft.md) requires one of the following:
+
+* Stopping the Raft network and starting the new network with the state at the time of migration.
+  That is, historical transactions and state history are lost.
+
+* Replaying the historical transactions on the new network.
```

Comment:
Do we describe anywhere how to do this? Might be worth linking too. Or perhaps raise create a ticket to add the missing content

Reply:
Antony says it depends on the user's specific network setup and situtation, so I think just advising to contact support is fine.